### ISSUE TITLE
Bad Navbar and Search UI  in mobile Devices.

### ISSUE THREAD

**Issue by ArnabBCA** [InternetHealthReport/ihr-website#686]
**Describe the bug**
1. The Navbar does not take the full width in mobile devices as a result the search bar is very small.
2. Also the Search Bar is not symmetric bad UI. 

**To Reproduce**

1. Go to 'https://ihr.iijlab.net/ihr/en-us'
2. Reduce Screen Size

**Expected behavior**
1. The Navbar should take the full width in mobile devices to make the search bar bigger.
2. Redesign Search bar According to the screenshot provided.

**Screenshots**
Default Navbar
![Redesign](https://github.com/InternetHealthReport/ihr-website/assets/113178195/9a39eeba-ca22-4ca5-a09b-0a4906bc250a)

**Additional context**
Redesigned Search Bar Example. **Note Final Search bar will match with the website color scheme**  
![Demo 1](https://github.com/InternetHealthReport/ihr-website/assets/113178195/69135ecc-9ecf-4c18-8069-f96c1030ca64)
![Demo 2](https://github.com/InternetHealthReport/ihr-website/assets/113178195/e9383749-ba4b-4248-88c0-f93087b1aa2f)

**Comment by ArnabBCA** [comment]
@mohamedawnallah I can solve this issue can you assign it :)

**Comment by romain-fontugne** [comment]
Sorry, you have too many issues assigned. Please revolve other issues first

**Comment by noobCoderVP** [comment]
Can you assign me this issue, I will fix the UI, I will add media queries to reduce font-size and box-sizes.

**Comment by YannyTwoo** [comment]
**Bug 1** - This doesn't seem to be an issue. 
**Bug 2** - Can address by removing the search box and just having a search icon, the search box can appear on click.

I'd like to work on this fix, can it be assigned to me?

Images for Bug 1:
![Screenshot 2024-01-21 at 10 06 17 PM](https://github.com/InternetHealthReport/ihr-website/assets/79789927/a464e572-6911-4ac8-bd6b-34312e303a32)
![IMG_1862](https://github.com/InternetHealthReport/ihr-website/assets/79789927/22ed537d-6087-45ff-a4bc-fc0ee93d08e1) 
![IMG_1861](https://github.com/InternetHealthReport/ihr-website/assets/79789927/45b017dc-2a15-4752-9d17-4fbd1d5f02ac)

**Comment by dpgiakatos** [comment]
The responsiveness of the site is not a priority, given its extensive information and complex charts that are better suited for larger screens. Therefore, implementing this change will require a redesign of our header (potentially incorporating the search in the toolbar). However, I believe it's essential to discuss this internally before proceeding with the change.

**Comment by pankaj87072** [comment]
@dpgiakatos can i do this one?

**Comment by nouralmulhem** [comment]
Hey, if this is to be solved can i tackle it? @dpgiakatos 
not sure if this is to be solved or to be discussed further

**Comment by dpgiakatos** [comment]
Please describe your plan to resolve this issue. If you intend to modify the UI, please provide some mockups to illustrate your proposed changes for discussion.

**Comment by nouralmulhem** [comment]
@dpgiakatos okay let me share my thoughts and validate the approach with you

let me first provide a screenshots for the different views

## Desktop
![image](https://github.com/InternetHealthReport/ihr-website/assets/76218033/c1aa52a9-76ba-4b65-81d4-d89bf5c1321b)

## Tablet
![image](https://github.com/InternetHealthReport/ihr-website/assets/76218033/5dba9d65-f931-4cce-a72a-cd7c84995bd6)

## Mobile 
![image](https://github.com/InternetHealthReport/ihr-website/assets/76218033/aad18cc5-9a6a-4260-96c1-0b43895f52ef)


so basically the idea i have now is at **(max-width: 1024px)** we give the `SearchBar` a `flex-grow 1` this will make this component expand on smaller screens without affecting neither the padding nor the other components

## Before:
![image](https://github.com/InternetHealthReport/ihr-website/assets/76218033/dc9496d4-007d-4387-a2b1-70e076e80d8b)


## After:
![image](https://github.com/InternetHealthReport/ihr-website/assets/76218033/7df4457d-798d-4901-a527-b278b09d26f6)

**Comment by dpgiakatos** [comment]
Your proposed solution looks good. You can start addressing it.
